### ISSUE TITLE
Manual polling feature

### ISSUE THREAD

**Issue by Rorkh** [protoncr/tourmaline#63]
Add method in Tourmaline::Poller to manually poll updates. 
It can be useful if you want to do other events in main loop (because I don't understand Crystal concurrency yet and I didn't find other way to do other things while bot is running).

I did it such way in my code but I have no time to make proper PR
```crystal
# Modification of Tourmaline::Poller with manual poll feature
require "tourmaline"

module Burservis
  class PollingNotStarted < Exception; end

  # The `Poller` class is responsible for polling Telegram's API for updates,
  # and then passing them to the `Dispatcher` for processing.
  class Poller
    Log = ::Log.for(self)

    getter offset : Int64

    def initialize(client : Tourmaline::Client)
      @client = client
      @offset = 0_i64
      @polling = false
    end

    def poll
      if !@polling
        raise PollingNotStarted.new("Polling is not started")
      end

      updates = @client.get_updates(offset: offset, timeout: 30)
      updates.each do |update|
        @client.dispatcher.process(update)
        @offset = Int64.new(update.update_id + 1)
      end
    end

    # Starts polling Telegram's API for updates.
    def start(manual_poll : Bool?)
      @client.delete_webhook
      Log.info { "Polling for updates..." }
      @polling = true

      if !manual_poll
        while @polling
          poll
        end
      end
    end

    def stop
      @polling = false
    end
  end
end

```

also could be nice to make poller getter in Client

**Comment by watzon** [comment]
So the whole reason behind making the `Poller` and `Server` classes was so that they could be used and modified outside of the context of the `Client` itself. Both are also just pretty thin wrappers around Telegram API actions. In the case of the `Poller`, that's `client.get_updates`.

That being said I can definitely  split the `poll` method into multiple methods so that you can poll for a single update manually. I won't be adding a getter to the `Client` class though, because you can just as easily create a `Poller` manually.

```crystal
client = Tourmaline::Client.new(...)
poller = Tourmaline::Poller.new(client)

# ...
```

**Comment by watzon** [comment]
Solved in 65ceaa5515a9e5a57a497f27c0f4bbaa2929de86
